src/Microsoft.AspNetCore.SignalR/HubContext.cs

```diff
@@ -20,6 +22,11 @@ public HubContext(HubLifetimeManager<THub> lifetimeManager)
 
         public virtual IGroupManager Groups { get; }
 
+        public IClientProxy AllExcept(List<string> excludedIds)
```

Comment:
Also, should have a `params string[] excludedIds` overload (possibly via an extension method) so you can call `.AllExcept(connection1, connection2, connection3)...`